stable/artifactory/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [10.0.2] - Jul 8, 2020
```

Comment:
@KierranM  Can you please bump the version here , CI is failing

Reply:
🤦 updated